### DISCUSSION TITLE
Support for Bright Sky (DWD)

### DISCUSSION THREAD

**Discussion by pablo03v** [breezy-weather/breezy-weather#694] [Q&A] [answered]
When choosing a data source the app redirected me to a help page about more info. It suggests the data source DWD but I can't find the option in the app. 

Is this not implemented yet? Where can I find it?

Thanks in advance!

**Comment by papjul** [comment, marked as answer]
Will be in next version. The documentation was completed as soon as it was implemented while it was still fresh in my memory.
We will auto-suggest sources based on country from next version, so we avoid sending user to this doc, which is not the most optimized user experience ;)

When in doubt about a feature implementation, you can check CHANGELOG of next version here: https://github.com/breezy-weather/breezy-weather/blob/main/CHANGELOG.md

**Reply by pablo03v** [reply]
Awesome then :)
